packages/console/src/monaco/MonacoTheme.module.scss

```diff
@@ -3,71 +3,72 @@
 
 :export {
   // iris dark theme
-  error-foreground: $red;
-  background: $content-bg;
-  foreground: $gray-100;
+  error-foreground: var(--dh-color-visual-red);
+  background: var(--dh-color-content-background);
+  foreground: var(--dh-color-gray-900);
   line-height: 19px; // 19 is the line height in the default monaco theme
 
-  //code rules
-  //more nuanced grays required in a few spots
-  string: $yellow;
-  string-delim: $gray-400;
-  delimiter: mix($gray-400, $gray-300, 30%);
-  predefined: $green;
-  keyword: $blue;
-  storage: $red;
-  number: $purple;
-  operator: $red;
-  identifier: $gray-100;
-  namespace-identifier: $red;
-  identifier-js: $yellow;
-  comment: mix($gray-500, $gray-400, 40%);
+  //code ruless
+  string: var(--dh-color-visual-yellow);
```

Comment:
Need to introduce semantic base variables for things like this. MonacoTheme.module.scss shoudn't be hard tied to things like "yellow" since that may differ from theme to theme.